CHANGELOG-4.0.md

```diff
@@ -1,5 +1,6 @@
 # [4.0.4](https://github.com/phalcon/cphalcon/releases/tag/v4.0.4) (2020-xx-xx)
 ## Added
+- Added a way to utilize GitHub actions to run database tests against each RDBMS and reworked the testing suite. [#14779](https://github.com/phalcon/cphalcon/issues/14779)
```

Comment:
I didn't add any comments about our Codeception 4 upgrade. Maybe we should add that here as well.